docker-compose.yml

```diff
@@ -19,16 +19,8 @@ services:
     volumes:
       - ${GITBASEPG_REPOS_FOLDER}:/opt/repos
   bblfsh:
-    image: "bblfsh/bblfshd:v2.7.2"
+    image: "bblfsh/bblfshd:v2.9.0-drivers"
     privileged: true
-    volumes:
-      - type: volume
-        source: drivers
-        target: /var/lib/bblfshd
-    entrypoint: ["/bin/sh"]
-    command:
-    - "-c"
-    - "bblfshd & sleep 5 && bblfshctl driver install --recommended && tail -f /dev/null"
 
 volumes:
```

Comment:
No need for the volumes section anymore?

Reply:
You are right, thanks. Fixed in f5224d1